pkg/discovery/dtofactory/node_entity_dto_builder_test.go

```diff
@@ -283,3 +280,75 @@ func Test_getAffinityCommoditiesSold(t *testing.T) {
 	assert.Equal(t, proto.CommodityDTO_LABEL, *commodities[0].CommodityType)
 	assert.Equal(t, pod, *commodities[0].Key)
 }
+
+func Test_getSuspendProvisionSettingByNodeType(t *testing.T) {
+	node1 := mockNode()
+	labels1 := make(map[string]string)
+	labels1["[k8s label] beta.kubernetes.io/os"] = "windows"
+	node1.SetLabels(labels1)
+	stitchingManager1 := stitching.NewStitchingManager(stitching.UUID)
+	stitchingManager1.StoreStitchingValue(node1)
+	nodeEntityDTOBuilder1 := NewNodeEntityDTOBuilder(metricsSink, stitchingManager1)
+	properties1, _ := nodeEntityDTOBuilder1.getNodeProperties(node1)
+
+	node2 := mockNode()
+	labels2 := make(map[string]string)
```

Comment:
no need for `make(map)`
see previous comment

Reply:
done